spec/builtin-fc.test.sh

```diff
@@ -0,0 +1,258 @@
+## tags: interactive
+## compare_shells: bash
+## oils_failures_allowed: 2
+
+#### fc -l
```

Comment:
Can you describe the tests in more detail?    `-l` is supposed to list the commands

Although actually I'm a bit confused in reading the test

I think it would be better not to use `seq 3` in the hist file?

Can we make it `echo 1; echo 2; echo 3` so it looks like a real history file?